tests/stage-test/src/daemon/computed-vars

```diff
@@ -3,3 +3,5 @@ Git branch: __ENV_[GIT_BRANCH]__
 Git commit: __ENV_[GIT_COMMIT]__
 Git clean: __ENV_[GIT_CLEAN]__
 Go arch: __ENV_[GO_ARCH]__
+RELEASE: __ENV_[RELEASE]__
```

Comment:
No need to put `RELEASE` here. This test is for computed vars only.

Reply:
Ok